lib/codelog/cli.rb

```diff
@@ -23,5 +23,12 @@ def release(version_number, release_date =
                 Date.today.strftime(Codelog::Config.date_input_format))
       Codelog::Command::Release.run version_number, release_date
     end
+
+    desc 'bump [VERSION_TYPE] <RELEASE_DATE>', 'Bumps the next version,
+     being it major, minor or patch'
+    def bump(version_type, release_date =
+                Date.today.strftime(Codelog::Config.date_input_format))
+      Codelog::Command::Bump.run version_type, release_date
```

Comment:
@Guilherme2112 the test coverage is decreasing just because of this line, it is a simple test to be made, I think its worth it for the :100: coverage.
Look for other tests like this in `spec/codelog/command/cli_spec.rb`

Reply:
@luisbevilacqua I see, fixed on 52f559.